multinode/bootstrapping.md

```diff
@@ -0,0 +1,191 @@
+# Data Node Bootstrapping [](distdb-bootstrap)
+
+Scaling up multi-node Timescale is achieved by adding data nodes to
+the access nodes using the `add_data_node` and `attach_data_node`
+functions. In order for a data node to be available to the access node
+it is necessary to ensure that it is properly setup to work as a data
+node. This process is refereed to as *bootstrapping the data node*.
+
+## Connections to the data node
+
+All connections to the data node are made as the current user on the
+access node. Which requires that:
+
+1. The current user exists on both the data node and access node.
```

Comment:
I believe it is important to mention which privileges the current user should have on the data nodes.

Reply:
These are the privileges required for connecting to the data node, but more are needed to actually bootstrap the data node.